<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSettingsPage</class>
 <widget class="QWidget" name="VideoSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Video settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,1">
      <item>
       <widget class="QRadioButton" name="shareRadio">
        <property name="text">
         <string>Share video settings with:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shareCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>or</string>
     </property>
     <property name="indent">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>heightEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="UnitLineEdit" name="heightEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="scalingCombo"/>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="heightLbl">
        <property name="text">
         <string>Max keyframe interval:</string>
        </property>
        <property name="buddy">
         <cstring>keyIntervalEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="nameLbl">
        <property name="text">
         <string>Canvas scaling:</string>
        </property>
        <property name="buddy">
         <cstring>scalingCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="fpsLbl">
        <property name="text">
         <string>Canvas scaling quality:</string>
        </property>
        <property name="buddy">
         <cstring>scalingQualityCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="scalingQualityCombo"/>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="widthLbl">
        <property name="text">
         <string>CPU usage:</string>
        </property>
        <property name="buddy">
         <cstring>cpuUsageCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="UnitLineEdit" name="widthEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="buddy">
         <cstring>widthEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="UnitLineEdit" name="keyIntervalEdit">
        <property name="placeholderText">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QComboBox" name="cpuUsageCombo"/>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="encoderCombo"/>
      </item>
      <item row="2" column="5">
       <widget class="UnitLineEdit" name="bitrateEdit"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Average bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>bitrateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encoder:</string>
        </property>
        <property name="buddy">
         <cstring>encoderCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="newRadio">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text">
            <string>Create new video settings:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="StyledInfoFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recommended video settings</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="bitrateCalcBtn">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Display video and audio settings calculator</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you need help figuring out what your video settings should be based on your internet connection we recommend that you use our calculator below. This calculator determines the highest quality settings that you can use while still maintaining maximum compatibility with network targets and your viewers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/unitlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>StyledFrame</class>
   <extends>QFrame</extends>
   <header>Widgets/styledframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StyledInfoFrame</class>
   <extends>QFrame</extends>
   <header>Widgets/styledframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>shareRadio</tabstop>
  <tabstop>shareCombo</tabstop>
  <tabstop>newRadio</tabstop>
  <tabstop>widthEdit</tabstop>
  <tabstop>heightEdit</tabstop>
  <tabstop>scalingCombo</tabstop>
  <tabstop>scalingQualityCombo</tabstop>
  <tabstop>encoderCombo</tabstop>
  <tabstop>bitrateEdit</tabstop>
  <tabstop>cpuUsageCombo</tabstop>
  <tabstop>keyIntervalEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
